assets/app/scripts/app.js

```diff
@@ -248,7 +248,16 @@ angular
         templateUrl: 'views/util/logout.html',
         controller: 'LogoutController'
       })
-
+      // legacy redirects
+      .when('/createProject', {
+        redirectTo: '/create-project'
```

Comment:
Hmm. I like the idea.  Searching, doesn't look like there is a simple way to run a function right in the routes block unless we don't do the redirect (could run a templateUrl function).  I'd prefer to keep the redirect until we remove.